website/content/docs/use-cases/hcp-hashistack-quickstart.mdx

```diff
@@ -0,0 +1,495 @@
+    Name = "${var.name}-public"
+  }
+
+  azs            = slice(data.aws_availability_zones.available.*.names[0], 0, 3)
+   enable_dns_hostnames            = true
+  enable_ipv6                     = true
+  assign_ipv6_address_on_creation = true
+  public_subnet_ipv6_prefixes     = [0, 1, 2]
+
+  tags = {
+    Terraform = "true"
+    Name      = var.name
+  }
+}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+```
+
+### HVN
+
+In the [`hcp.tf`](https://github.com/xiaolin-ninja/2048_hcp_hashistack/blob/main/hcp.tf) configuration file,
+we define a HashiCorp Virtual Network (HVN) named "xx-2048" and peer it to the VPC.
+HVNs enable you to deploy HashiCorp Cloud products without having to manage the networking details.
+Creating peering connections from your HVN will allow you to connect and launch AWS resources from your HCP account.
+Each peering connection requires an HVN route to set its CIDR block.
+
+Once the VPC is peered with the HVN, the resources deployed inside the VPC will be able to reach each other.
+
+```terraform
+resource "hcp_hvn" "xx_2048_hvn" {
+  hvn_id         = "xx-2048"
+  cloud_provider = "aws"
+  region         = var.region
+}
+
+resource "hcp_aws_network_peering" "peer" {
+  hvn_id          = hcp_hvn.xx_2048_hvn.hvn_id
+  peering_id      = "hcp-hashistack"
+  peer_vpc_id     = module.vpc.vpc_id
+  peer_account_id = module.vpc.vpc_owner_id
+  peer_vpc_region = var.region
+}
+
+resource "hcp_hvn_route" "hvn-to-vpc" {
+  hvn_link         = hcp_hvn.xx_2048_hvn.self_link
+  hvn_route_id     = "hvn-to-vpc"
+  destination_cidr = module.vpc.vpc_cidr_block
+  target_link      = hcp_aws_network_peering.peer.self_link
+}
+
+resource "aws_vpc_peering_connection_accepter" "peer" {
+  vpc_peering_connection_id = hcp_aws_network_peering.peer.provider_peering_id
+  auto_accept               = true
+  accepter {
+    allow_remote_vpc_dns_resolution = true
+  }
+}
+
+resource "aws_route" "hvn-peering" {
+  route_table_id            = module.vpc.public_route_table_ids[0]
+  destination_cidr_block    = hcp_hvn.xx_2048_hvn.cidr_block
+  vpc_peering_connection_id = aws_vpc_peering_connection_accepter.peer.id
+```
+
+## Define HCP Consul
+
+HCP Consul enables you to quickly deploy Consul servers in AWS while offloading the operations burden to the SRE experts at HashiCorp.
+
+Provisioning HCP Consul is incredibly easy.
+You can define the cluster in Terraform using the [`hcp_consul_cluster`](https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/resources/consul_cluster) resource,
+as shown in `hcp.tf`:
+
+```terraform
+resource "hcp_consul_cluster" "xx_2048_consul" {
+  hvn_id          = hcp_hvn.xx_2048_hvn.hvn_id
+  cluster_id      = "xx-2048-consul"
+  tier            = "development"
+  public_endpoint = true
+}
+```
+
+You can also provision a Consul cluster with one click in the HCP UI.
+
+## Define HCP Vault
+
+Defining a Vault cluster in Terraform is as simple as with Consul,
```

Comment:
Unsure how to word this, but trying to avoid the "simple" language:
```suggestion
Defining an HCP Vault cluster via Terraform also requires minimal configuration,
```